OpenRA.Mods.Common/UtilityCommands/ActorStatsExport.cs

```diff
@@ -79,7 +80,9 @@ public static DataTable GenerateTable()
 							var burst = weapon.Burst;
 							var delay = weapon.BurstDelay;
 							var damage = warhead.Damage;
-							var damagePerSecond = (1000f / Game.Timestep) * (damage * burst) / (delay + burst * rateOfFire);
+
+							// Assumes the default game speed (25 ticks / sec)
+							var damagePerSecond = (1000f / 40) * (damage * burst) / (delay + burst * rateOfFire);
```

Comment:
I'm not prepared to add core engine code just to work around a utility rule that its not clear whether anyone uses.